ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -26,7 +26,7 @@ rule align:
         insertions="data/insertions.csv",
         translations="data/translations.zip",
     params:
-        translations=lambda w: "data/translations/{gene}.fasta",
+        translations="data/translations/{gene}.fasta",
```

Comment:
Hmm, this change currently causes can error: 
```
WildcardError in rule align in file /nextstrain/build/workflow/snakemake_rules/nextclade.smk, line 20:
Wildcards in params cannot be determined from output files. Note that you have to use a function to deactivate automatic wildcard expansion in params strings, e.g., `lambda wildcards: '{test}'`. Also see https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules:
'gene'
```

Reply:
For context, this param is written this way due to the [requirements of the nextclade `--output-translations` option](https://docs.nextstrain.org/projects/nextclade/en/stable/user/output-files.html#aligned-peptides).

Reply:
Oh, thanks for catching! I didn't realize that this change was on ingest which is untested in CI. Will fix.

Reply:
Yeah, sorry to have mislead you with my [previous comment](https://github.com/nextstrain/monkeypox/pull/161#discussion_r1270973818). 

Reading through https://github.com/snakemake/snakemake/issues/2166 made me realize that's it might be safest to use `lambda w:` for params that might have curly braces in the value.

Reply:
Well that's kinda annoying and unintuitive, but now it makes sense why there are so many seemingly unused lambda functions scattered around. Time to put this in our ✨[Snakemake style guide](https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html)✨!

Reply:
Fixed in [force-push](https://github.com/nextstrain/monkeypox/compare/da64839aeca114b5daae4b8191779266e7bc0748..927ad6cdf0f7e96384ab8a53f87aee7b5c4e658b).